larky/src/main/java/com/verygood/security/larky/modules/crypto/CryptoChaseModule.java

```diff
@@ -0,0 +1,75 @@
+package com.verygood.security.larky.modules.crypto;
+
+import com.google.common.collect.ImmutableList;
+import com.verygood.security.larky.modules.vgs.crypto.ChaseCrypto;
+import java.util.List;
+import java.util.ServiceLoader;
+import lombok.SneakyThrows;
+import net.starlark.java.annot.Param;
+import net.starlark.java.annot.ParamType;
+import net.starlark.java.annot.StarlarkBuiltin;
+import net.starlark.java.annot.StarlarkMethod;
+import net.starlark.java.eval.StarlarkBytes;
+
+@StarlarkBuiltin(
+    name = "Chase",
+    category = "BUILTIN",
```

Comment:
This module looks like it belongs in the VGS namespace, not built-in or vendors.